improved message design(#52)

LalitNM opened the pull request:
@abhigyank I fixed #52 and my compiler also formatted these files :sweat_smile:

LalitNM committed:
Improved message design!

Fixes #52.

abhigyank commented on public/client.js:
```diff
@@ -92,45 +102,57 @@ socket.on('user join', function(data) {
+    // Format Message
+    const div = document.createElement('div');
+    const username = document.createElement('small');
+    const TimeStamp = document.createElement('small');
```
Can you change `TimeStamp` to `timestamp` ? I think that is fine. CapWords in JS looks weird, probably suited in Python.

LalitNM replied:
Done!

LalitNM committed:
Removed timestamp at start of conversation

Now timestamp will be visible in every message, so we don't need a separate timestamp at start of conversation

abhigyank commented:
Merged, Good work @LalitNM :rocket:
